get_github_coderabbit_review/coderabbit_fetcher/analyzers/metadata_enhancer.py

```
@@ -0,0 +1,444 @@
+        distribution_parts = []
+        for file_type, count in file_counts.most_common():
+            distribution_parts.append(f"{file_type}: {count}")
+
+        return ", ".join(distribution_parts) if distribution_parts else "mixed files: 1"
+
+    def _calculate_priority_distribution(self, comment_counts: Dict[str, int]) -> str:
+        """Calculate priority distribution based on comment types."""
+        # Map comment types to priorities based on expected output
+        actionable = comment_counts.get("actionable", 0)
+        nitpick = comment_counts.get("nitpick", 0)
+        outside_diff = comment_counts.get("outside_diff", 0)
```

Ruff (0.13.2)

Local variable `outside_diff` is assigned to but never used

Remove assignment to unused variable `outside_diff`

(F841)